packages/fast-usdc/contract/quickSend.flows.js

```diff
@@ -0,0 +1,186 @@
+ *   fundingPool: OrchestrationAccountI;
+ *   feeAccount: OrchestrationAccountI;
+ * }} QuickSendAccounts
+ */
+
+/**
+ * @satisfies {OrchestrationFlow}
+ * @param {Orchestrator} orch
+ * @param {{
+ *   terms: QuickSendTerms & StandardTerms;
+ *   chainHub: GuestInterface<ChainHub>;
+ *   t?: ExecutionContext<{ nextLabel: Function }>; // XXX
+ *   makeSettleTap: (accts: QuickSendAccounts) => Guarded<{
+ *     receiveUpcall: (event: VTransferIBCEvent) => void;
+ *   }>;
+ *   makeWatcherCont: (accts: QuickSendAccounts) => {
+ *     invitationMakers: InvitationMakers;
+ *     actions: Record<string, Function>;
+ *   };
+ * }} ctx
+ * @param {ZCFSeat} seat
+ * @param {{}} _offerArgs
+ */
+export const initAccounts = async (orch, ctx, seat, _offerArgs) => {
+  const { log = console.log } = ctx.t || {};
+
+  const agoric = await orch.getChain('agoric');
+  const aInfo = await agoric.getVBankAssetInfo();
+  for (const a of aInfo) {
+    if (a.brand === ctx.terms.brands.USDC) {
+      const [baseName, baseDenom] = ['noble', 'uusdc']; // ???
+      await orch.getChain(baseName);
+      ctx.chainHub.registerAsset(a.denom, {
+        baseDenom,
+        baseName,
+        chainName: 'agoric',
+        brand: ctx.terms.brands.USDC,
+      });
+      break;
+    }
+  }
+
+  const fundingPool = await agoric.makeAccount();
+  const settlement = await agoric.makeAccount();
+  const feeAccount = await agoric.makeAccount();
+  const accts = harden({ fundingPool, settlement, feeAccount });
+  const tap = ctx.makeSettleTap(accts);
+  // @ts-expect-error tap.receiveUpcall: 'Vow<void> | undefined' not assignable to 'Promise<any>'
+  const registration = await settlement.monitorTransfers(tap);
+
+  log('@@@what to do with registration?', registration);
+
+  const cont = ctx.makeWatcherCont(accts);
+  /** @type {ResolvedContinuingOfferResult} */
+  const watcherFacet = harden({
+    publicSubscribers: {
+      fundingPool: (await fundingPool.getPublicTopics()).account,
+      settlement: (await settlement.getPublicTopics()).account,
+      feeAccount: (await feeAccount.getPublicTopics()).account,
+    },
+    ...cont,
+  });
+
+  seat.exit();
+  return watcherFacet;
+};
+harden(initAccounts);
+
+/**
+ * TODO: move to a method on ChainHub
+ *
+ * @param {string} value
+ * @returns {ChainAddress}
+ */
+const asChainAddress = value => {
+  // TODO: from ChainInfo, from chain-registry
+  const chain = {
+    prefix: 'dydx',
+    chainId: 'dydx-mainnet-1',
+  };
+  assert(value.startsWith(chain.prefix), 'unsupported prefix');
+  const { chainId } = chain;
+  return harden({ encoding: 'bech32', value, chainId });
+};
+
+/**
+ * @param {Orchestrator} _orch
+ * @param {{
+ *   terms: QuickSendTerms & StandardTerms;
+ *   t?: ExecutionContext<{ nextLabel: Function }>; // XXX
+ * }} ctx
+ * @param {QuickSendAccounts & Passable} accts
+ * @param {import('./client-types.js').CCTPTxEvidence} offerArgs
+ */
+export const handleCCTPCall = async (_orch, ctx, accts, offerArgs) => {
```

Comment:
Is it worth stubbing a notifier/subscriber for the Oracle, or adding a comment that denotes this behavior?

Reply:
elaborate? I don't understand what you mean.

Reply:
I wanted us to represent Oracle Feed in a separate exo, but probably OK to punt on for now